### DISCUSSION TITLE
AI 竟會種族歧視！清大攜手跨國團隊揭露「醫學倫理隱憂」

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#662] [科技新知]
AI 人工智慧竟會種族歧視？清華大學資工系助理教授郭柏志與麻省理工學院、哈佛大學跨國研究發現，當採用先進的 AI 深度學習演算法來判讀 X 光片及電腦斷層掃描時，電腦竟會從醫學影像分辨出患者是黑人、白人或亞洲人，並影響判斷的準確率，但科學家至今還不明白 AI 是如何辦到。
究團隊中一名哈佛大學教授表示，當團隊成員稱 AI 可以從 X 光片來判斷人種，自己簡直不敢置信，認為這真的太瘋狂，一定是哪裡出錯，但當越來越多的證據浮現，有團隊成員則直言感覺很「恐怖」。
https://technews.tw/2022/06/09/medical-ethics-concerns/

**Comment by HsuChen88** [comment]
誤判率會不會其實只是由不同人種生理結構上的不同而產生，或許不是他在"歧視"?

**Comment by Lai27330** [comment]
可能是因為不同人種的樣本數不一，加上生活環境不太相似導致構造有細微差異，才導致AI判讀後得到不同的結果呢?

**Comment by HaoFish123** [comment]
感謝分享，但我覺得AI種族歧視，還在人類的預料範圍之內。

**Comment by treeann** [comment]
其實哪裡都會有歧視 我們能做的就是管好自己

**Comment by sys511613** [comment]
希望研究團隊能找出造成的原因，或許真的有可能是先天上體格的差異造成誤判也說不定

**Comment by yucheng1028** [comment]
種族歧視無所不在阿

**Comment by etin2468** [comment]
這應該是導入的數據庫的問題了吧?

**Comment by Lynn12378** [comment]
感覺應該是其他技術層面的問題

**Comment by Hchejxnc** [comment]
感覺起來像是數據庫的原因，導致AI誤判率的差距，但判斷人種說不定是天生的差異?

**Comment by Jimmy911218** [comment]
我覺得看起來並沒有歧視的問題?

**Comment by CPCpyt** [comment]
我覺得只是判斷誤差的問題，與其扣帽子進行責備，
能不能讓大家一起討論如何改進，促進正向發展。

**Comment by ruthchen1020** [comment]
我有發過相關的文章，也是在討論人工智慧倫理上的隱憂。
https://github.com/NCU-GS4719-Python/Python-Community/discussions/888#discussion-4636157

**Comment by StevenWanglolz** [comment]
我看過類似的ted talk，可能是在訓練時，樣本數不一的關係吧

**Comment by Doreen331** [comment]
不只在看x光片時會有「歧視」的現象產生，之前在上另一門課時就有聽到，其實像siri之類的語音辨識也有歧視的現象，但最根本的原因應該還是跟當初在訓練時投入的樣本數有關，只是跟種族相關被扣上了歧視的字眼

**Comment by peggy0329** [comment]
這種新聞標題真的很會誤導人...
我認為誤判率不同根跟樣本數有關
而判別AI可判別不同人種可能是精密的技術可以偵測到人類肉眼看不出來的差異
文章也沒有證實AI會有歧視，不應該直接這樣武斷的下文章標題吧

**Comment by swerfg03joe** [comment]
原文 discrimination雖然有歧視的意思但更多的是表達區別。
像這篇文章的作者相當不專業以為用聳動的標題博眼球才是正途。
說到底原本人種就有區別，為了自身的政治利益將區分膚色打成種族歧視是相當沒道理的行為。
既然AI只是看圖就能高正確率地區別人種，不就提供學者另一種研究人體的方向嗎?

**Comment by BlahFish1006** [comment]
覺得很正常，沒有什麼歧視的問題，因為不同性別、不同人種的生理構造本來就不同。
這就好比拿「以A居多的訓練集」訓練的模型，用「以B居多的測試集」去測試它，在正常情況下當然對於B的準確率會低、A準確率會高。

**Comment by Sylvie0601** [comment]
其實我也認為因為人種不同的關係，生理結構、基因這些都不宜樣而形成的AI解讀，感覺不能因此而認為AI歧視不同人種吧!我感覺這種應該只能算是辨識不同人種，歧視應該要有辨別之後對不同種族做不公平的動作感覺才應該可以稱為歧視。

**Comment by harrylee1971** [comment]
不太可能吧?X光片打下去剩下骨頭，感覺應該是有其他的因素才發現說可以是黑人，或是純粹是造假新聞而已
